docs/testcafe-testing-library/intro.md

```diff
@@ -0,0 +1,61 @@
+---
+id: intro
+title: Testcafe Testing Library
+---
+
+[`testcafe-testing-library`][gh] allows the use of dom-testing queries within
+[Testcafe](https://devexpress.github.io/testcafe/) cross-browser end-to-end web
+testing.
+
+```
+npm install --save-dev testcafe testcafe-testing-library
+```
+
+- [testcafe-testing-library on GitHub][gh]
+
+## Usage
+
+`testcafe-testing-library` provides custom Selectors allowing you to query the
+dom.
+
+Add `testcafe-testing-library` to your test fixture's `beforeEach` hook:
+
+```javascript
+import {
+  getByText, //or any other queries you want
+  addTestcafeTestingLibrary,
+} from 'testcafe-testing-library'
+
+fixture`selectors`.beforeEach(addTestcafeTestingLibrary)
+  .page`http://localhost:13370`
+```
+
+You can now import & use `getBy`, `getAllBy`, `queryBy` and `queryAllBy`
+selectors in your tests.
+[See `dom-testing-library` API for reference](dom-testing-library/api-queries.md)
```

Comment:
Should probably mention that these queries are all pre-bound to `document.body` so people don't assume they have to pass the `container` argument.